cmd/reconciler/ip.go

```diff
@@ -5,11 +5,30 @@ import (
 	"flag"
 	"github.com/k8snetworkplumbingwg/whereabouts/pkg/storage"
 	"os"
+	"strings"
 
 	"github.com/k8snetworkplumbingwg/whereabouts/pkg/logging"
 	"github.com/k8snetworkplumbingwg/whereabouts/pkg/reconciler"
 )
 
+func knownErrorCase(err error) bool {
```

Comment:
we may have comment for the function to mention that it is used for accept some failure case as valid case.

Reply:
Agreed, added a comment :+1: